### DISCUSSION TITLE
Organizing torrents

### DISCUSSION THREAD

**Discussion by Dutchlikeme** [transmission/transmission#5458] [Q&A]
Is there a way to add separators or tags to torrent files? Make it so I can't delete it until I go into properties 
and untick *CANNOT DELETE UNLESS UNTICKED* 

Let's say I have a torrent for a show with multiple seasons and multiple single movies
At the moment the torrents get added sorted through ABC so the one torrent of the show gets lost in the sea of movies
Could I make the torrent static in the list, so I don't accidentally delete it when deleting the movie torrents?
Is there a way to organize this in the Windows client?

**Comment by dantearmok** [comment]
Transmission currently has no such property.

How 'bout creating a group for that/those torrents?  For me, groups work very well for organizing entries.

**Reply by Dutchlikeme** [reply]
Where can I create a group?
![image](https://user-images.githubusercontent.com/100804523/234952961-5d0b3427-af23-40b6-81ca-cdcce3b7da86.png)

**Reply by dantearmok** [reply]
(Sorry; I'm flying a bit blind; I'm used to the Mac client)

There should be a "Use Groups" type item, in a View menu.  Then the Groups themselves are editable in the Settings.

**Reply by GaryElshaw** [reply]
That's a macOS-only feature, currently.

**Reply by NelsonSR** [reply]
Click on the "Set Location" option on that menu and create or select a folder according to some criteria. In Transmission, there's no such thing as events associated with folders, filenames, or tags.

**Reply by Dutchlikeme** [reply]
> Click on the "Set Location" option on that menu and create or select a folder according to some criteria. In Transmission, there's no such thing as events associated with folders, filenames, or tags.

Yeah I've got separate folders for my files, it's just the application that isn't sorted

**Reply by NelsonSR** [reply]
Have you tried [Transmission Remote GUI](https://sourceforge.net/projects/transgui/)? Buggy but much more featured than the official clients.

**Reply by Dutchlikeme** [reply]
> Have you tried [Transmission Remote GUI](https://sourceforge.net/projects/transgui/)? Buggy but much more featured than the official clients.

I'll give it a try, thanks!

Edit, I've tried to install it but changed my mind as I had to install alot of other things with it (rebuild it or something like that?)

Edit2, I've managed to install it (didn't use the easy install method, read the website for once and found the downloads) but I can't get it working

**Reply by NelsonSR** [reply]
I have no idea. I've been using it on Debian for some months. Any dependency is solved by APT (Advance Packaging Tool). So, no problems at all with the installation, only with the software per se in some cases, like closing the program.

**Comment by dantearmok** [comment]
oOo.  Is there any equiv in the client the OP is using?

**Reply by GaryElshaw** [reply]
The Qt client functionality is similar to the macOS View --> Sort options.

**Comment by MatanZ** [comment]
> Is there a way to add separators or tags to torrent files? Make it so I can't delete it until I go into properties and untick _CANNOT DELETE UNLESS UNTICKED_

The easiest way to implement this is using labels. In the torrent remove method, make sure it does not have this specific label. Then you can use the already available label UI to set torrents deletable/undeleteable. Or the client may add a special UI for that (which will set/clear this label in the background).

**Reply by Dutchlikeme** [reply]
How do I set a label?

**Reply by MatanZ** [reply]
That depends on the UI you use. 

`transmission-remote` has the `-L` option to set labels.

`tremc`  uses the keys 'b', 'B', '^L' to add, set, or remove labels.

**Comment by Dutchlikeme** [comment]
I've been trying to get transmission-remote working and have failed, I just want 2 features, labels and dark mode.
